tensorflow/compiler/xla/service/gpu/cudnn_batchnorm_thunk.cc

```diff
@@ -30,6 +30,47 @@ namespace gpu {
 
 namespace dnn = se::dnn;
 
+namespace {
+void IsInputOutputDtypeValid(const HloInstruction* hlo) {
```

Comment:
- Since the function is imperative it should be called `CheckXYZ` not `IsXYZ`.
 - Dtype is a TF-ism.  In XLA it is called "primitive type".